Integrations/MicrosoftGraphListener/MicrosoftGraphLIstener.yml

```diff
@@ -0,0 +1,385 @@
+category: Email Gateway
+commonfields:
+  id: MicrosoftGraphListener
+  version: -1
+configuration:
+- display: ID (received from the authorization step - see Detailed Instructions (?)
+    section)
+  name: auth_id
+  required: true
+  type: 4
+- display: Refresh token (received from the authorization step - see Detailed Instructions
+    (?) section)
+  name: refresh_token
+  required: true
+  type: 4
+- display: Key (received from the authorization step - see Detailed Instructions (?)
+    section)
+  name: enc_key
+  required: true
+  type: 4
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Email address from which to fetch incidents  (e.g. "from:example@demisto.com")
+  name: mailbox_to_fetch
+  required: true
+  type: 0
+- defaultvalue: Inbox
+  display: Name of the folder from which to fetch incidents (supports Folder ID and
+    sub-folders e.g. Inbox/Phishing)
+  name: folder_to_fetch
+  required: true
+  type: 0
+- defaultvalue: 15 minutes
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  name: first_fetch_interval
+  required: false
+  type: 0
+- defaultvalue: '50'
+  display: Maximum number of emails to pull per fetch.
+  name: emails_fetch_limit
+  required: false
+  type: 0
+description: Microsoft Graph lets your app get authorized access to a user's Outlook
+  mail data in a personal or organization account.
+display: Microsoft Graph Mail Listener And Sender
+name: MicrosoftGraphListener
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: A CSV list of email addresses for the 'to' field.
+      isArray: true
+      name: to
+      required: true
```

Comment:
Should I raise an error from the code in case of empty recipients input or error received from api is clear enough: `Error in API call [400] - Bad Request
{'error': {'code': 'ErrorInvalidRecipients', 'message': "At least one recipient isn't valid., A message can't be sent because it contains no recipients.", 'innerError': {'request-id': '1981c11a-7930-4027-adee-666b64a442c3', 'date': '2019-11-12T13:49:50'}}}` ?